fast/stages-multitenant/0-bootstrap-tenant/README.md

```diff
@@ -85,7 +85,7 @@ As shown in the script output above, the provider file is a template used as a s
 
 Note that the `outputs_location` variable is disabled by default, you need to explicitly set it in your `terraform.tfvars` file if you want output files to be generated by this stage. This is a sample `terraform.tfvars` that configures it, refer to the [org-level bootstrap stage documentation](../../stages/0-bootstrap/README.md#output-files-and-cross-stage-variables) for more details:
 
-```hcl
+```tfvars
```

Comment:
weird, I did not know tfvars was supported; it's still hcl though, any benefit in changing this?

Reply:
It's mostly cosmetic: you don't have to add `tftest skip` to prevent that snippet from being parsed as a test